fdb-record-layer-core-shaded/fdb-record-layer-core-shaded.gradle

```diff
@@ -102,7 +102,4 @@ publishing {
             artifact tasks.shadedJavadocJar
         }
     }
-    bintray {
-        publications = ["shadow"]
```

Comment:
It's possible we'll need to add an `artifactoryPublish` section here that specifies it should use the `shadow` publication. The artifacts for 2.10.178.0 look right to me, so I'm hesitant to change this until we know it's a problem, though: https://ossartifacts.jfrog.io/artifactory/fdb-record-layer/org/foundationdb/fdb-record-layer-core-shaded/2.10.178.0/